### PR TITLE
test: add cloudbuild-test.yaml that runs as presubmit

### LINKED ISSUE #102: Add presubmit for verifying that the cloudbuild will run
We can duplicate the cloudbuild.yaml with the exception of the images: section. This can test to see that the cloudbuild execution will complete, but won't publish the images.

### PR DESCRIPTION
**Pull request by chingor13**
These triggers are sync'ed by the sync-triggers.sh script.

The new cloudbuild-test.yaml files are the same as the cloudbuild.yaml files except they don't push the images at the end of a successful build.

Fixes #102

### CODE REVIEW

File: elixir/cloudbuild-test.yaml [hunk 1]
```diff
@@ -0,0 +1,113 @@
+# Copyright 2018 Google LLC
```
Comment (codyoss) [inline elixir/cloudbuild-test.yaml:1]: nit: 2020. Other files as well

File: .kokoro/sync-triggers.sh [hunk 2]
```diff
@@ -63,3 +63,30 @@ do
...
+  gcloud beta builds triggers create github \
+    --project="${project}" \
+    --repo-name="${repoName}" \
+    --repo-owner="${repoOwner}" \
+    --description="Build testing docker images in ${directory}" \
+    --included-files="${directory}/**" \
+    --name="${triggerName}" \
+    --pull-request-pattern="${branch}" \
+    --build-config="${config}"
+done
```
Comment (codyoss) [inline .kokoro/sync-triggers.sh:92]: nit: return

**Review by busunkim96 — APPROVED** [review]

**Review by codyoss — APPROVED** [review]

**Review by JustinBeckwith — APPROVED** [review]

**Review by lesv — APPROVED** [review]

File: java/cloudbuild-test.yaml [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  - name: gcr.io/cloud-builders/docker
+    args:
+      [
+        "tag",
+        "gcr.io/$PROJECT_ID/java10",
+        "gcr.io/cloud-devrel-public-resources/java10",
+      ]
+    waitFor: ["java10-build"]
+
+  # Java 11 build
```
Comment (lesv) [inline java/cloudbuild-test.yaml:93]: We should probably get a Java 15 build here as well.

**Review by bcoe — APPROVED** [review]

**Review by dazuma — APPROVED** [review]
OK for Ruby/Elixir.

**Pull request merged**